src/validation/mod.rs

```diff
@@ -30,7 +31,7 @@ pub struct ValidationInfo<'bytecode> {
     pub(crate) globals: Vec<Global>,
     #[allow(dead_code)]
     pub(crate) exports: Vec<Export>,
-    pub(crate) func_blocks: Vec<Span>,
+    pub(crate) func_blocks_sidetables: Vec<(Span, Sidetable)>,
```

Comment:
I think we can keep the name `func_blocks` and use a docstring to explain that we also attach the sidetable

Reply:
Per function sidetables are going to be replaced with per module sidetables anyway.